.travis.yml

```diff
@@ -1,27 +1,36 @@
-language: python
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
+# The only current way of using OS X virtual machines in Travis CI
+# is setting up the environment for Objective-C
+# This is not a problem because Python will be installed through conda
+
+language: objective-c
```

Comment:
I think it happens because it expects some kind of version of objective-c as you used with python.

Sorry, I was so focused in **Anaconda** and **Travis** that I forgot to add the `pip install -e .` line in `appveyor.yml` to match the erasing of the `__init__.py` in the `tests`folder. If you read [the article](https://pytest.org/latest/goodpractises.html) it is a good mess the problem with the paths 😱. It'll be fixed within a minute!